cmd/release-controller/sync_test.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/diff"
+	"k8s.io/test-infra/prow/github"
+)
+
+func TestGetBugzillaPRs(t *testing.T) {
```

Comment:
Ah. Is there any way for us to get at the structures that are used to generate the changelog instead of parsing it post-facto with regex?

Reply:
Some of the structures (`InfoOptions`, `func NewInfoOptions`, `func InfoOptions.LoadReleaseInfo`) are exposed, but the ones that actually do the diffs and changelog aren't (`calculateDiff`, `releaseDiffContentChanges`, `commitsForRepo`, etc). We could copy over this code, but I thought that wouldn't be ideal (though regex-ing the generated changelog isn't the greatest thing either).